nucleus/common/internal-api/src/main/java/org/glassfish/internal/deployment/Deployment.java

```diff
@@ -173,20 +175,20 @@
 
     public interface DeploymentContextBuilder {
```

Comment:
You can also remove public from this inner interface.

Reply:
Static can be removed from the inner class ApplicationDeployment (classes within interfaces are always static)